data/json/items/ranged/spearguns.json

```diff
@@ -203,5 +203,30 @@
       [ "underbarrel mount", 1 ]
     ],
     "pocket_data": [ { "pocket_type": "MAGAZINE", "ammo_restriction": { "fishspear": 1 } } ]
+  },
+  {
+    "id": "powerhead",
+    "type": "GUN",
+    "name": { "str": "powerhead" },
+    "description": "A single-shot firearm mounted on a stick.  Commonly used for spear fishing or for defense against sharks.  This one has an activation pin the must be pulled to fire.",
+    "weight": "4000 g",
+    "volume": "4765 ml",
+    "longest_side": "165 cm",
+    "looks_like": "pipe",
+    "price": 8500,
+    "range": 2,
```

Comment:
This needs to be negative to lower the range of shooting, it's a modifier to the range granted by the ammo you load it with(00 shot is 12, dragon's breath is 6, slugs are 24).

Reply:
Ah, OK. I intended the shooting range to be 2 tiles like a spear.
So I'll try -10 as a start.